e2e-tests/typescript-nodenext-resolution/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
what about `"private": true`?

Reply:
it doesn't matter, lerna setup to publish only from `./packages`

Reply:
anyway, make it consistent. The `e2e-tests/node-api/package.json` has this flag